Bellerophon/Bellerophon/Sources/BPConfig.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  BellerophonConfig.swift
+//  Bellerophon
+//
+//  Created by Satinder Singh on 11/17/17.
+//
+
+import UIKit
+
+/// Configuration for Bellerophon
+public struct BellerophonConfig {
+
+    /// The root window
+    public var window: UIWindow
+
+    /// View to be shown when the kill switch is turned on. Must be set by the application using Bellerophon
+    public var killSwitchView: UIView?
+
+    /// View to be shown when force updated is required. Must be set by the application using Bellerophon
+    public var forceUpdateView: UIView?
+
+    /// Bellerophon manager delegate
+    public weak var delegate: BellerophonManagerDelegate?
+
+    /// Initalizer for `BellerophonManager`
+    /// - Note: It is possible to supply no views and still handle force update and killswitch by utilzing methods found in
+    ///         `BellerophonManagerDelegate` via `shouldForceUpdate` and / or `shouldKillSwitch`
+    /// - Parameters:
+    ///   - window: Root UIWindow
+    ///   - killSwitchView: UIView to display when kill switch is triggered. If nil, no view will be shown for kill switch event.
+    ///   - forceUpdateView: UIView to display when force update is triggered. If nil, no view will be shown for force update event.
+    ///   - delegate: `BellerophonManagerDelegate` with optional callbacks.
+    public init(window: UIWindow, killSwitchView: UIView?, forceUpdateView: UIView?, delegate: BellerophonManagerDelegate?) {
```

Comment:
default params for killSwitchView, forceUpdateView and delegate so that a client which isn't using them can just call `init(window:)`

Reply:
No one would use `Bellerophoron` with `init(window:) ` because it means they are not using any of its functionality. It would keep it clean for the developers if they choose to not use all of them so I suppose I can default nil them all